documentation/docs/getting-started/index.md

```diff
@@ -5,149 +5,121 @@ path: /getting-started/
 
 # Getting started
 
-Fusion.js is a web application framework for building high quality universal applications. It provides a modular architecture with a strong focus on testability and maintainability.
+Fusion.js, Uber’s open source universal web framework, is designed to make web development easier and produce lightweight,
```

Comment:
Minor: What is going on with the line breaks here?

Reply:
I may have been a tad aggressive line breaking this file. No other file does this though so I'll remove the aggro line breaking.